.travis.yml

```diff
@@ -1,21 +1,17 @@
 language: node_js
 node_js:
-- '8'
-- '10'
-- '11'
+  - "8"
+  - "10"
+  - "11"
 script:
-- yarn add --dev axios@^${AXIOS_VERSION} && yarn test
+  - yarn add --dev axios@^${AXIOS_VERSION} && yarn test
 after_success: yarn coverage
 before_install: yarn global add greenkeeper-lockfile@1
 before_script: greenkeeper-lockfile-update
 after_script: greenkeeper-lockfile-upload
 env:
-- AXIOS_VERSION=v0.13
-- AXIOS_VERSION=v0.14
-- AXIOS_VERSION=v0.15
-- AXIOS_VERSION=v0.16
-- AXIOS_VERSION=v0.17
-- AXIOS_VERSION=v0.18
+  - AXIOS_VERSION=v0.18
```

Comment:
I was trying to be a kind as possible and avoid breaking old versions. Maybe we could have a habit of keeping the last 5, or something like that?